src/gui/static/src/app/app-routing.module.ts

```diff
@@ -0,0 +1,89 @@
+import { NgModule } from '@angular/core';
+import { WalletsComponent } from './components/pages/wallets/wallets.component';
+import { SendSkycoinComponent } from './components/pages/send-skycoin/send-skycoin.component';
+import { RouterModule } from '@angular/router';
+import { PendingTransactionsComponent } from './components/pages/settings/pending-transactions/pending-transactions.component';
+import { OutputsComponent } from './components/pages/settings/outputs/outputs.component';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/skycoin/skycoin/pullRequest?prid=7163600)
```suggestion
import { OutputsComponent } from "./components/pages/settings/outputs/outputs.component";
```